app/rules/v2_to_v3/routing_rules.py

```diff
@@ -0,0 +1,22 @@
+def routing_conversions(schema):
+
+    all_blocks = schema.blocks
+
+    for block in all_blocks:
+        update_radio_question_condition_in_routing_rules(block, schema)
+
+
+def update_radio_question_condition_in_routing_rules(item, schema):
+
+    if "routing_rules" in item:
+        for rule in item["routing_rules"]:
+            whens = rule.get("goto").get("when")
```

Comment:
```suggestion
            whens = rule.get("goto", {}).get("when")
```

Don't chain two `get` methods without defaulting to an object that implements it in the first call.